Sources/ArcGISToolkit/Components/OfflineMapAreasView/PreplannedMapModel.swift

```diff
@@ -0,0 +1,31 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import ArcGIS
+
+/// An object that encapsulates state about a preplanned map.
+public class PreplannedMapModel: ObservableObject, Identifiable {
+    /// The preplanned map area.
+    @Published var preplannedMapArea: PreplannedMapArea
```

Comment:
Is there a benefit to make this variable published here? I'm assuming you did it so when it changes an UI update can happen. But I can't think of an example yet.